### PR TITLE
rentRecord and returnRecord: dto, service, facade

### PR DESCRIPTION
**Pull request by jakubsarmir**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -7,6 +7,11 @@
             <artifactId>validation-api</artifactId>
             <version>1.1.0.Final</version>
         </dependency>
+        <dependency>
+            <groupId>cz.fi.muni.pa165</groupId>
+            <artifactId>carpark-persistence</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
```
Comment (jkuchar) [inline api/pom.xml]: No, there must be no dependency on persistance.

File: api/src/main/java/cz/fi/muni/pa165/api/facade/ReturnRecordFacade.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     * @param r is new return record
+     */
+    void create(ReturnRecordDTO r);
+
+    /**
+     * Method delete return record from db.
+     * @param r is return record to delete
+     */
+
+
```
Comment (jkuchar) [inline api/src/main/java/cz/fi/muni/pa165/api/facade/ReturnRecordFacade.java]: tu něco chybí ;)

File: service/src/main/java/cz/fi/muni/pa165/service/RentRecordServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    @Override
+    public List<RentRecord> getRecordsCreatedBetween(Date from, Date to) {
+        return rentRecordDao.getRecordsCreatedBetween(from, to);
+    }
+
+    @Override
+    public void create(RentRecord r) {
+
+    }
```
Comment (rtrembecky) [inline service/src/main/java/cz/fi/muni/pa165/service/RentRecordServiceImpl.java]: tu by nieco malo byt

File: api/src/main/java/cz/fi/muni/pa165/api/facade/RentRecordFacade.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    /**
+     * Method to retrieve rent record with corresponding id.
+     * @param id id of rent record
+     * @return return rent record with certain id
+     */
+    public RentRecordDTO findById(UUID id);
+
+    /**
+     * Method to retrieve rent records with corresponding car.
+     * @param c certain car
```
Comment (rtrembecky) [inline api/src/main/java/cz/fi/muni/pa165/api/facade/RentRecordFacade.java]: toto je uz carId ako pozeram, takze prerobit komentare trochu

File: service/src/main/java/cz/fi/muni/pa165/service/ReturnRecordServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+    @Override
+    public List<ReturnRecord> findByUser(UUID userId) {
+        return returnRecordDao.findByUser(userId);
+    }
+
+    @Override
+    public List<ReturnRecord> getRecordsCreatedBetween(Date from, Date to) {
+        return null;
+    }
```
Comment (rtrembecky) [inline service/src/main/java/cz/fi/muni/pa165/service/ReturnRecordServiceImpl.java]: pridat implementaciu

File: service/src/main/java/cz/fi/muni/pa165/service/ReturnRecordServiceImpl.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+    @Override
+    public void create(ReturnRecord r) {
+
+    }
```
Comment (rtrembecky) [inline service/src/main/java/cz/fi/muni/pa165/service/ReturnRecordServiceImpl.java]: implementacia

**Pull request merged**